comp/metadata/inventorychecks/README.md

```diff
@@ -37,6 +37,7 @@ The payload is a JSON dict with the following fields
     - `state` - **dict of string**: the current state of the log source.
       - `status` - **string**: one of `pending`, `error` or `success`.
       - `error` - **string**: the error description if any.
+    - `integration_name` - **string**: the name of the integration, can be empty.
```

Comment:
Could you also add `service`, `source`, and `tags`, as it seems they are missing ?